Source transfer impact on connected destination (#5575)
segmentio/segment-docs: src/guides/usage-and-billing/account-management.md

To move a source between workspaces, navigate to the source's **Settings** tab, then click **Transfer to Workspace**. Choose the workspace you're moving the source to, then click **Transfer Source**.

> When a source is transferred from one workspace to another, it will include all the connected destinations with the same settings.

Reviewer: ```suggestion
When you transfer a source from one workspace to another, the new source includes all the connected destinations with the same settings.
```
